### PR TITLE
Add provider for lmgtfy.com

### PR DESCRIPTION
**Pull request by akb**
`s -p lmgtfy passive aggressive behavior`

### CODE REVIEW

File: providers/lmgtfy/lmgtfy.go [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+	"github.com/zquestz/s/providers"
+)
+
+func init() {
+	providers.AddProvider("lmgtfy", &LMGTFYProvider{})
+}
+
+// LMGTFYProvider implements the Provider interface.
+type LMGTFYProvider struct{}
```
Comment (zquestz) [inline providers/lmgtfy/lmgtfy.go]: Check out the latest providers for the new naming convention. I think you will like it. =)

File: providers/lmgtfy/lmgtfy.go [hunk 2]
```diff
@@ -0,0 +1,20 @@
+package google
```
Comment (zquestz) [inline providers/lmgtfy/lmgtfy.go]: this should be lmgtfy

Comment (akb) [inline providers/lmgtfy/lmgtfy.go]: *blinks* yep. surprised that this compiled and ran fine.

File: providers/lmgtfy/lmgtfy.go (hunk 1 above)
Comment (akb) [inline providers/lmgtfy/lmgtfy.go]: Indeed. Using packages for what they're supposed to be used for...

On that note, there's really no need to export Provider, and then the comment can be deleted too.

Comment (zquestz) [inline providers/lmgtfy/lmgtfy.go]: True that would be a nice cleanup.

**Pull request merged**